app/views/view-claim.njk

```diff
@@ -32,7 +32,7 @@
       <h1 class="govuk-caption-l">Claim by <a class="govuk-link" href="/agreement/{{applicationReference}}/claims?page=1&returnPage=view-claim&reference={{reference}}">{{ title }}</a> </h1>
     </div>
     <div class="govuk-grid-column-one-third govuk-!-text-align-right govuk-!-padding-left-0 govuk-!-padding-right-0">
-      <div class="govuk-tag govuk-!-margin-top-1 govuk-!-margin-bottom-3 {{ status.tagClass }}">{{ status.capitalisedtype }}</div>
+      <div class="govuk-tag govuk-!-margin-top-1 govuk-!-margin-bottom-3 govuk-!-text-align-left {{ status.tagClass }}" style="max-width: 300px;">k{{ status.normalType }}</div>
```

Comment:
Does this work for all tags? I know that some of the tags are stored as all capitals in the database

Reply:
is not directly for DB ,  normalType is a transformation -->  normalType: upperFirstLetter(formatStatusId(claim.statusId).toLowerCase())